source/test/sgp_mode_test/HealthHost.test.cc

```diff
@@ -9,6 +9,112 @@
 // Also test with health host with NOT, give it just barely enough CPUs to finish, check whether it manages to complete NOT
 // Trickier for mutualist, check just before enough CPUs and it should manage to finish
 
+
+TEST_CASE("Health host with symbiont loses/gains cycle 50% of time", "[kw]") {
+  emp::Random random(10);
+  
+  //TODO: The random number seed doesn't seem to be working, different values for the same seed
+
+  SymConfigSGP config;
+  config.SEED(10);
+  config.ORGANISM_TYPE(HEALTH);
+  config.STRESS_TYPE(PARASITE);
```

Comment:
STRESS_TYPE setting should be moved down into the "When parasites" section for cleanliness